tests/test_fetcher.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+
+"""
+<Program>
+  test_download.py
+
+<Author>
+  Teodora Sechkova tsechkova@vmware.com
+
+<Started>
+  December 18, 2020.
+
+<Copyright>
+  See LICENSE-MIT OR LICENSE for licensing information.
+
+<Purpose>
+  Unit test for 'fetcher.py'.
+"""
+# Help with Python 2 compatibility, where the '/' operator performs
+# integer division.
+from __future__ import division
+
+import logging
+import os
+import io
+import sys
+import unittest
+import tempfile
+import math
+
+import tuf
+import tuf.unittest_toolbox as unittest_toolbox
```

Comment:
Nit: should this be sorted alphabetically? `unittest_toolbox` below `requests_fetcher`

Reply:
Fixed in be399d2.